apps/playground-api/src/bots/GovBot.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiClient } from '@defichain/jellyfish-api-core'
+import { TokenResult } from '@defichain/jellyfish-api-core/src/category/token'
+import { Injectable } from '@nestjs/common'
+import { Interval } from '@nestjs/schedule'
+
+@Injectable()
+export class GovBot {
+  constructor (readonly client: ApiClient) {
+  }
+
+  @Interval(6000)
+  async run (): Promise<void> {
+    const tokens = await this.client.token.listTokens()
+    for (const id of Object.keys(tokens)) {
+      if (tokens[id].isLoanToken) {
+        await this.enablePayback(id, tokens)
+      }
```

Comment:
I think we should only enable a few scenarios to keep it saner. Easier for @thedoublejay to design different scenarios without this capability too.

Reply:
yea done